applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -93,11 +106,8 @@ static const char *get_dvfs_frequency_setting_name(enum dvfs_frequency_setting s
 static const char *get_dvfs_state_name(enum dvfs_state state)
 {
 	switch (state) {
-	case DVFS_STATE_INITIALIZING: return "DVFS_STATE_INITIALIZING";
-	case DVFS_STATE_LLPM_CONNECTED: return "DVFS_STATE_LLPM_CONNECTED";
-	case DVFS_STATE_USB_CONNECTED: return "DVFS_STATE_USB_CONNECTED";
-	case DVFS_STATE_CONFIG_CHANNEL: return "DVFS_STATE_CONFIG_CHANNEL";
-	case DVFS_STATE_SMP_TRANSFER: return "DVFS_STATE_SMP_TRANSFER";
+	FOR_DVFS_STATE_LIST(GET_ENUM_NAME)
```

Comment:
So that the syntax does not look a bit strange:

```suggestion
	FOR_DVFS_STATE_LIST(GET_ENUM_NAME);
```

Reply:
we will have then 2 semicolons as a result, but I guess it doesn't cause a problem.